doc/readme.html

```diff
@@ -504,6 +513,16 @@ <h4>Optional interfaces to other software packages</h4>
                 </p>
             </dd>
 
+            <dt><a name="Kokkos"/>
+                <a href="https://github.com/kokkos/kokkos" target="_top">Kokkos</a></dt>
+            <dd>
+                <p>
+                    <a href="https://github.com/kokkos/kokkos" target="_top">Kokkos</a> implements a programming model in C++ for writing performance portable applications targeting all major HPC platforms.
+                    To use a self compiled version, pass <code>-DKOKKOS_DIR=/path/to/kokkos</code> to the deal.II CMake call. deal.II does not support the version of <a href="https://github.com/kokkos/kokkos" target="_top">Kokkos</a>
```

Comment:
Is that true for the *current* version on github, or will this always be the case?

Reply:
The problem is that the CMake configure files are different when you use Kokkos alone and when you use Kokkos from Trilinos. So you need some kind of switch in `FindKOKKOS` to make this work and I really don't want to do it right now. To answer your question, it will always be the case until we improve our `FindKOKKOS`.

Reply:
But doesn't that mean that if we use a version of Trilinos that uses Kokkos internally, then we can't use the new class because the new class requires an external Kokkos that would then conflict with the one inside Trilinos?

Or would our cmake files pick up the Trilinos Kokkos in that case, and you're only adding this code here in case someone wants to use ArborX without Trilinos, and so we need an external Kokkos?

Reply:
> But doesn't that mean that if we use a version of Trilinos that uses Kokkos internally, then we can't use the new class because the new class requires an external Kokkos that would then conflict with the one inside Trilinos?

That's right, there could be a conflict but the CPU code in Kokkos is pretty stable so the two versions should be compatible

Reply:
But that's almost certainly going to lead to problems if we link against *both* a Trilinos-internal Kokkos version and an external Kokkos version at the same time, wouldn't it? I mean, there will be symbol clashes, resource problems, etc.

Reply:
That's possible, I haven't tried. Do you want me to add a comment about it?

Reply:
I don't see a good solution to this. Why not require that (i) if you want to use ArborX, you also need to configure for Trilinos, and (ii) if you configure for Trilinos, it needs to contain Kokkos?

Reply:
Alright I will fix the `FindKOKKOS` I wrote to also work with Trilinos's Kokkos :sob: